src/services/service-factory.test.tsx

```diff
@@ -4,9 +4,9 @@ import React from "react";
 import { act, render } from "@testing-library/react";
 import { ServiceFactory } from "./service-factory";
 import { CoreUtils } from "andculturecode-javascript-core";
-import { StubResourceRecord } from "../tests/stubs/stub-resource-record";
+import { StubResourceRecord } from "andculturecode-javascript-core";
 import { FactoryType } from "../tests/factories/factory-type";
-import mockAxios from "../tests/mocks/mock-axios";
+import mockAxios from "andculturecode-javascript-core/dist/tests/mocks/mock-axios";
```

Comment:
Is this how this needs to be imported? Directly from `dist/...`?

Reply:
No, I thought I found all of the named exports per your PR, but I missed this one 🤦 so gotta loop back at some point

Reply:
Fixed in my branch on Core for this issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/issues/3